### DISCUSSION TITLE
Cashier + Stripe:  subscription created on Stripe sets field "name" to  value of "default"

### DISCUSSION THREAD

**Discussion by myregistration** [laravel/framework#44224] [Q&A]
How is Cashier setting the subscription table field "name" to "default" when a subscription is created from within Stripe?  It creates the subscription table records correctly, except the name of the product shouldn't be "default".  I was using a different subscription name within the Laravel code.   Thanks!

**Comment by alkhachatryan** [comment]
Just saw this question, as I have it now too.
The answer in this briliant method :D
![image](https://github.com/laravel/framework/assets/22774727/f367bf4f-0f1a-4519-b24a-6f71f9b0a76d)

The webhook controller tries to get the name of the subscription from metadata, if it's not set - it gets default :D
![image](https://github.com/laravel/framework/assets/22774727/ea95d560-0401-4de0-8973-4f1dcb7ec9a4)


So you need to override the webhook controller and the method. For that you need to put the webhook route in your router and pass to your customer webhook controller


And this is what I actually did:
![image](https://github.com/laravel/framework/assets/22774727/2d5cc4f1-198a-4fe5-8f32-f334e67e6b5e)
